obal/data/roles/diff_package/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+- block:
```

Comment:
What is this block for?

Reply:
Check the `when` condition at the bottom now.

Reply:
yeah. btw, I think now the block does not  need to be a block, as it only has one task in it.